src/main/java/io/swagger/entity/media/MediaInfo.java

```diff
@@ -0,0 +1,30 @@
+package io.swagger.entity.media;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class MediaInfo {
```

Comment:
I haven't looked in to see how different this will be from the TitleData object. I think that any List object (ratings, actor list, genre list) may be tricky to store in a database (it could be stored as a string in a column).